### PR TITLE
Details pop up window

### PR DESCRIPTION
**Pull request by Drew246**
Added popup modal.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
...
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
+     </ul>
+     <button class="btn-default portfolio-items-detail">See project</button>
+   </div>
+   <div class="card-left-desk">
```
Comment (AlphaNtihinduka) [inline app.js:102]: - [ ] As suggested in the ️[project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), kindly try as much as possible to stick to the Figma original design. For this case, I suggest that you consider displaying all the contents of the popup card as seen in the Figma design ☺️. 
- [ ] In addition, I suggest that you reduce the blur effect to ensure that the background contents are blurry seen 👍.

Figma design 😎 
image

Your design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team🙋‍♂️
Good job working on your popup. You have done the design to the best expectation and incredibly well. 👏  
 
You still need to implement a few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Popup disappears when (x) is clicked ✔️ 
- Correct use of map function ✔️ 
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: app.js [hunk 2]
```diff
@@ -12,14 +14,146 @@ function toggleMobileMenu() {
...
+     <ul class="cano">
+       <li>${portfolioItemsDetail[i].title}<i class="fa fa-circle"></i></li>
+       <li>${portfolioItemsDetail[i].stack}<i class="fa fa-circle"></i></li>
+       <li>2015</li>
+     </ul>
+     <p>${portfolioItemsDetail[i].text}</p>
+     <ul class="lang">
+       ${portfolioItemsDetail[i].workStack
+    .map((stack) => `<li>${stack}</li>`)
+    .join('')}
```
Comment (Shedrack-Sunday) [inline app.js:98]: - [ ] You did a great job 👏 . However, you intriduced a new bug, and still need to fix the chnages suggested by the last reviewer 👍 
- Firstly, your cancel button has been omitted.

### View:
image

### Figma design:
image

- [ ] Secondly, just as the last reveiver pointed out, kindly adjust your popup. The buttom part should be visible on the screen, and fixed just as it is on figma. You can make reference  the materils for this week's for that 👍 

### Figma:
image

### Your design:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Drew246        👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the JavaScript features for interaction✔️ 
    
 
  
# Required Changes  🌵  

- [ ] You've done well, However, kindly check some key point note I pointed out to meet the requirements before moving to the next project 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Drew246** [comment]
Hi, I am not sure if something went wrong but I am not seeing the bug you
are talking about. My close button was visible when I checked. also, the
bottom of the modal is visible.
Can yo take another look?

**Comment by Shedrack-Sunday** [comment]
Hello @Drew246, this is the view I get from cloning your project locally. Please confirm the changes see are pushed to the current branch

**Comment by Drew246** [comment]
I pushed. Are you on the Details -popup branch?

**Comment by Shedrack-Sunday** [comment]
Yes, I used the branch in which you created the PR. Please snap the current popup that you have on that branch

**Comment by Drew246** [comment]
The current popup is already there. Something is wrong. Should I create a
new pull request? Can you please kindly verify the branch once more? I
double checked and everything is fine on my end.

**Comment by Drew246** [comment]
I have to sleep now. But i would like to zoom call you in a few hours. I do
not want to waste one of my reviews.

**Review by V-Blaze — APPROVED** [review]
Hi @Drew246  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Pop-up works as expected 💯
-  Project section generated dynamically👍
-  Good communication with previous reviewers 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**